doc/config.rst

```diff
@@ -0,0 +1,670 @@
+"""""""""""""""""""""""""
+
+    * ``eff_area`` = *float_value* (default = RUBIN_AREA) Collecting area of telescope in cm^2. Default: Rubin value from https://confluence.lsstcorp.org/display/LKB/LSST+Key+Numbers
+
+
+SkyLevel Value Type
+^^^^^^^^^^^^^^^^^^^
+
+Once the Rubin sky-model has been specified you can use the calculated sky level in other parts of the YAML file. An example is shown below:
+
+.. code-block:: yaml
+
+    image:
+        type: LSST_Image
+
+        sky_level: { type: SkyLevel }  # Computed from input.sky_model.
+        apply_sky_gradient: True
+
+
+Atmospheric PSF
+----------------
+
+The class is used to create the PSF which is induced by the atmosphere.  There are two parametric PSFs available: a double Gaussian and a Kolmogorov PSF. The ``AtmosphericPSF`` type is a fully ray-traced turbulent atmosphere with multiple atmospheric layers.  Additionally, you can optionally add a parametric PSF screen which simulates the Rubin Optics.
+
+.. _AtmosphericPSF-label:
+
+Key Name: atmosphericPSF
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+This keyword enables an atmospheric PSF with 6 randomly generated atmospheric screens.  Photons are raytraced through this atmosphere to produce a realistic atmospheric PSF.
+
+.. warning::
+    You should not attempt to use the option to add parametric optics (through the ``doOpt`` option) if you are using fully ray-traced optics.  Otherwise, you will simulate the optics twice.  See See :ref:`the stamp keyword <stamp-label>` below how to activate the ray-traced mode.
+
+
+Required keywords to set:
+""""""""""""""""""""""""""
+
+    * ``airmass`` = *float_value* (default =  None)  The aimass in the direction of the pointing.
+    * ``rawSeeing`` = *float_value*  The FWHM seeing at zenith at 500 nm in arc seconds
+    * ``band`` = *str_value* The filter band of the observation.
+    * ``boresight`` = *RaDec_value* The CelestialCoord of the boresight of the observation.
+
+
+Optional keywords to set:
+"""""""""""""""""""""""""
+
+    * ``t0`` = *float_value* (default = 0.0) Exposure time start in seconds.
+    * ``exptime`` = *float_value*  (default = 30.0) Exposure time in seconds.
+    * ``kcrit`` = *float_value* (default = 0.2) Critical Fourier mode at which to split first and second kicks.
+    *  ``screen_size`` = *float_value* (default = 819.2) Size of the phase screens in meters.
+    *  ``screen_scale`` = *float_value* (default = 0.1) Size of phase screen "pixels" in meters.
+    *  ``doOpt`` = *bool_value* (default = True) Add in optical phase screens? *SEE WARNING ABOVE*
+    *  ``nproc`` = *int_value* (default = None)  Number of processes to use in creating screens. If None (default), then allocate one process per phase screen, of which there are 6, nominally.
+    *  ``save_file`` = *str_value* (default = None) A file name to use for saving the built atmosphere.  If the file already exists, then the atmosphere is read in from this file, rather than being rebuilt.
+
+
+Key Name: DoubleGaussianPSF
```

Comment:
Hmm... well you have the choice here to specify either an atmosphericPSF, DoubleGaussianPSF, or KolmogorovPSF all in the same place (unless I am mis-understanding something). I n the case of the later two you need to explicitly set the the FWHW etc.  

So, in some ways it seems weird  that the AsmosphericPSF is here too, but I think that is because it can read in an input file so these naturally go along with it.